modules/migration-configuring-cors-3.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// migration/migrating_openshift_3_to_4/migrating-openshift-3-to-4.adoc
+[id='migration-configuring-cors-3_{context}']
+= Configuring cross-origin resource sharing on an {product-title} 3 cluster
+
+You must configure cross-origin resource sharing on the {product-title} 3 cluster to enable communication between the CAM tool on the target cluster and the source cluster's API server.
+
+.Procedure
+
+. Log in to the {product-title} 4 cluster.
+. Obtain your CAM tool URL:
++
+----
+$ oc get -n openshift-migration route/migration -o go-template='(?i}//{{ .spec.host }}(:|\z){{ println }}' | sed 's,\.,\\.,g'
```

Comment:
There is a subtle typo here of '}' when we wanted ')', the typo was found in the upstream mig-operator docs and recently updated.

Please change to:
$ oc get -n openshift-migration route/migration -o go-template='(?i)//{{ .spec.host }}(:|\z){{ println }}' | sed 's,\.,\\.,g'


Note, in the original:  go-template='(?i}//
We want:                   go-template='(?i)//